react/rollup.config.ts

```diff
@@ -0,0 +1,58 @@
+import * as path from 'path'
+import resolve from '@rollup/plugin-node-resolve'
+import commonjs from '@rollup/plugin-commonjs'
+import typescript from 'rollup-plugin-typescript2'
+import json from '@rollup/plugin-json'
+import packageJson from './package.json'
+
+const plugins = [
+    json(),
+    resolve({
+        preferBuiltins: false,
+        mainFields: ['module', 'main', 'jsnext:main', 'browser'],
+        extensions: ['.js', '.jsx', '.ts', '.tsx'],
+    }),
+    typescript({
+        useTsconfigDeclarationDir: true,
+    }),
+    commonjs(),
+]
+
+/**
+ * Configuration for the ESM build
+ */
+const buildEsm = {
+    external: ['posthog-js', 'react', 'react-dom'],
```

Comment:
`react-dom` is currently only used within tests, so I don't believe defining it as an external for the build serves a purpose here. Am I correct in assuming this is a bit of future-proofing for potential React components added down the line?

Reply:
Removed it